### PR TITLE
add instagram support and icon

### LINKED ISSUE #98: Instagram support
I was missing the Instagram icon when using this theme.

### PR DESCRIPTION
**Pull request by patrickderitter**
Fixes #98

### CODE REVIEW

**Review by seemethere — APPROVED** [review]
LGTM

File: layouts/partials/footer.html [hunk 1]
```diff
@@ -37,6 +37,12 @@
             </a>
         {{ end }}
 
+        {{ if .Site.Params.instagram }}
+            <a href="https://www.instagram.com/{{ .Site.Params.linkedin }}" target="_blank">
```
Comment (mtn) [inline layouts/partials/footer.html]: I think you mean `.Site.Params.instagram` (rather than linkedin). I've made this mistake before too.

Comment (mtn) [reply layouts/partials/footer.html]: Otherwise, LGTM

Comment (patrickderitter) [reply layouts/partials/footer.html]: @mtn you’re right.. see, this is why we can’t have nice things. Thanks for fixing this error!

**Pull request merged**